Makefile

```diff
@@ -204,7 +230,7 @@ $(COREPROTO):
 
 grpc: | $(COREPROTO)
 
-hash luacartesi grpc test lint coverage-report check-format format check-format-lua check-lua format-lua:
+hash luacartesi grpc lint check-format format check-format-lua check-lua format-lua:
```

Comment:
`check-lua` and `format-lua` are not considering lua files in `tests` directory anymore, we may need to move them to this Makefile.

I think the same is happening for `lint` and `format` with C++ code in `tests/misc`.

Reply:
ok will take a look.